File: src/main/java/kr/co/vacgom/persistence/global/util/UuidUtility.java
package kr.co.vacgom.persistence.global.util;

import com.github.f4b6a3.uuid.UuidCreator;

import java.util.UUID;

public class UuidUtility {

    public static UUID generateRandomUUID() {
        return UuidCreator.getTimeOrderedEpochPlus1();
    }
}

File: vacgom-api/src/main/kotlin/kr/co/vacgom/api/global/util/UuidUtility.kt
package kr.co.vacgom.api.global.util

import com.github.f4b6a3.uuid.UuidCreator
import java.util.*

object UuidUtility {

    fun generateRandomUuid(): UUID {
        return UuidCreator.getTimeOrderedEpochPlus1()
    }
}
